captcha/events.py

```diff
@@ -60,3 +60,25 @@ async def on_member_join(self, member: Member):
     @commands.Cog.listener()
     async def on_member_remove(self, member: Member):
         await self.cleaner(member)
+
+    @commands.is_owner()
+    @commands.command()
+    async def captcha(self, ctx, *members: Member):
+        """Start a captcha challenge for the specified user"""
+        await ctx.send(
+            "The challenge has been launched for all members of the server specified in the command.\n"
+            + "```"
+            + ctx.message.content
+            + "```"
+        )
```

Comment:
We can eradicate these lines entirely. Instead, we can just add a notice message and add typing until the final message has been sent.

```suggestion
        await ctx.send("Running Captcha challenges... this may take a while!")
        await ctx.trigger_typing()
```